app/src/main/java/com/example/robin/roomwordsample/Adapter/WordListAdapter.kt

```diff
@@ -124,5 +141,18 @@ class WordListAdapter internal constructor(
         notifyItemChanged(position)
         wordViewModel.insert(word)
     }
+
+    private fun showKeyboard() {
```

Comment:
Create a utlis.kt file under the Utils folder and add the showKeyboard() and closeKeyboard() in that class and use it access these functions from anywhere in the app.

Reply:
> Create a utlis.kt file under the Utils folder and add the showKeyboard() and closeKeyboard() in that class and use it access these functions from anywhere in the app.
Sure.

Reply:
Done.